tests/integration/profiles/five-safes-crate/test_5src_10_outputs.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) 2024-2025 CRS4
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from rocrate_validator.models import Severity
+from tests.ro_crates import ValidROC
+from tests.shared import do_entity_test, SPARQL_PREFIXES
+
+# set up logging
+logger = logging.getLogger(__name__)
+
+
+# ----- SHOULD fails tests
+
+
+def test_completed_createaction_does_not_have_result():
+    """
+    Test a Five Safes Crate where `CreateAction` has `CompleteActionStatus` but
+    does not have the property `result`.
+    (We remove `result` from `CreateAction` if this has `CompleteActionStatus`)
+    """
+    sparql = (
+        SPARQL_PREFIXES
+        + """
+        DELETE {
+            ?action schema:result ?o .
+        }
+        WHERE {
+            ?action a schema:CreateAction ;
+                      schema:actionStatus "http://schema.org/CompleteActionStatus" .        }
+        """
+    )
+
+    do_entity_test(
+        rocrate_path=ValidROC().five_safes_crate_request,
```

Comment:
Use the "result" crate for these tests - the request crate's CreateAction doesn't reference any `result`s so won't trigger the relevant checks

```suggestion
        rocrate_path=ValidROC().five_safes_crate_result,
```

Reply:
Done.